src/chains.ts

```diff
@@ -190,6 +214,10 @@ export const L1_STANDARD_BRIDGE_INFORMATION: Record<
       l2Chain: 'redstone',
       l1StandardBridgeAddress: '0xc473ca7E02af24c129c2eEf51F2aDf0411c1Df69',
     },
+    {
+      l2Chain: 'metall2',
+      l1StandardBridgeAddress: '0xAEb9E3dD0074A8dd19065f55235591c3Ac0c413E',
```

Comment:
this doesn't match whats in the superchain-registry https://github.com/ethereum-optimism/superchain-registry/blob/10ff8b6e7b57cbd894d333d13c59c08bb3970cac/superchain/configs/mainnet/metal.toml#L49